Adds option to disable emoji

frmendes opened the pull request:
I don't know if this project is still active but I added an option to disable the emoji. Just pass the `-e` flag and it should work. It fallsback to `spark`, which wasn't happening on OS X.

Goles commented:
Hey @frmendes it's active, haven't really tried this out. You still using Battery?

frmendes commented:
Hey @Goles. Yes, I use it daily on tmux. It's supposed to resolve #29. Here's a screenshot:

<img width="762" alt="screen shot 2017-01-25 at 20 45 40" src="https://cloud.githubusercontent.com/assets/4325027/22308526/49161b3e-e33f-11e6-9f18-263fe02147ce.png">

The version that's running, however, is the one I forked so it's a bit outdated. I can rebase off master and see if any issues popped up. I can also change the flag or behaviour if you want, I'm open to suggestions.

Goles commented:
I would like to merge your changes in :)

So if you can do that it would be great.

PS: Please update the docs if that’s needed too.

Best,
Nico
> On Jan 25, 2017, at 12:47 PM, Fernando Mendes <notifications@github.com> wrote:
> 
> Hey @Goles <https://github.com/Goles>. Yes, I use it daily on tmux. It's supposed to resolve #29 <https://github.com/Goles/Battery/issues/29>. Here's a screenshot:
> 
>  <https://cloud.githubusercontent.com/assets/4325027/22308526/49161b3e-e33f-11e6-9f18-263fe02147ce.png>
> The version that's running, however, is the one I forked so it's a bit outdated. I can rebase off master and see if any issues popped up. I can also change the flag or behaviour if you want, I'm open to suggestions.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/Goles/Battery/pull/27#issuecomment-275228152>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AALEXn9M6rh3VFo0tX7zT9i7KTxHLI5Qks5rV7TcgaJpZM4JS8Pa>.
>

frmendes commented:
Hi @Goles 

Just rebased off `master`. I also squashed changes into a single commit. Here's a preview of the output:

<img width="738" alt="screen shot 2017-01-26 at 09 56 09" src="https://cloud.githubusercontent.com/assets/4325027/22327183/d92c900e-e3ad-11e6-9e05-a66418199c05.png">

Goles commented:
Thanks @frmendes
